### PR TITLE
Explain how to run job on nested groups

### PR DESCRIPTION
**Pull request by adamlazik1**
https://bugzilla.redhat.com/show_bug.cgi?id=2209987

* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.8/Katello 4.10
* [x] Foreman 3.7/Katello 4.9 (planned Satellite 6.14)
* [x] Foreman 3.6/Katello 4.8
* [x] Foreman 3.5/Katello 4.7 (Satellite 6.13)
* [ ] Foreman 3.4/Katello 4.6 (EL8 only)
* [ ] Foreman 3.3/Katello 4.5 on EL7 & EL8 (Satellite 6.12 on EL8 only; orcharhino 6.4/6.5 on EL8 only)
* [ ] Foreman 3.2/Katello 4.4 on EL7 & EL8
* [ ] Foreman 3.1/Katello 4.3 on EL7 & EL8 (Satellite 6.11 EL7/8; orcharhino 6.3 on EL7/8)
* We do not accept PRs for Foreman older than 3.1.

### CODE REVIEW

File: guides/common/modules/proc_executing-a-remote-job.adoc [hunk 1]
```diff
@@ -10,6 +10,17 @@ To use the CLI instead of the {ProjectWebUI}, see the xref:cli-executing-a-remot
 . Select the *Job category* and the *Job template* you want to use, then click *Next*.
 . Select hosts on which you want to run the job.
 If you do not select any hosts, the job will run on all hosts you can see in the current context.
++
+[NOTE]
+====
+If you want to select a host group and all of its subgroups, it is not sufficient to select the host group as the job would only run on hosts directly in that group and not on hosts in subgroups.
+Instead, you must either select the host group and all of its subgroups or use this search query:
```
Comment (maximiliankolb) [inline guides/common/modules/proc_executing-a-remote-job.adoc:17]:
```suggestion
Instead, you must either select the host group and all of its subgroups or use this search query:

```

File: guides/common/modules/proc_executing-a-remote-job.adoc [hunk 2]
```diff
@@ -10,6 +10,17 @@ To use the CLI instead of the {ProjectWebUI}, see the xref:cli-executing-a-remot
...
+[options="nowrap" subs="+quotes,verbatim,attributes"]
+----
+hostgroup_fullname ~ _My_Host_Group_/*
```
Comment (maximiliankolb) [inline guides/common/modules/proc_executing-a-remote-job.adoc:20]:
```suggestion
hostgroup_fullname ~ _My_Host_Group_*
```

Tested on Foreman 3.8: Only if I drop the trailing slash I get all hosts within the parent host group and sub host groups.

**Review by maximiliankolb — CHANGES_REQUESTED** [review]
foreman_3_8_search_query
foreman_3_8_host_groups

Can you reproduce this?

File: guides/common/modules/proc_executing-a-remote-job.adoc (hunk 2 above)
Comment (adamlazik1) [reply guides/common/modules/proc_executing-a-remote-job.adoc]: Confirmend and applied.

File: guides/common/modules/proc_executing-a-remote-job.adoc (hunk 1 above)
Comment (adamlazik1) [reply guides/common/modules/proc_executing-a-remote-job.adoc:17]: Applied.

**Review by mjivraja — APPROVED** [review]
I liked how you simplified the sentence structure and formation of line `16`. :green_heart: 
LGTM, style-wise.

Also, please rebase for the link-checker fix before merging.

**Comment by adamlazik1** [comment]
Thanks @mjivraja. rebased. @maximiliankolb is this gtg on your end?

File: guides/common/modules/proc_executing-a-remote-job.adoc [hunk 3]
```diff
@@ -10,6 +10,18 @@ To use the CLI instead of the {ProjectWebUI}, see the xref:cli-executing-a-remot
...
+[NOTE]
+====
+If you want to select a host group and all of its subgroups, it is not sufficient to select the host group as the job would only run on hosts directly in that group and not on hosts in subgroups.
+Instead, you must either select the host group and all of its subgroups or use this search query:
+
+[options="nowrap" subs="+quotes,verbatim,attributes"]
+----
+hostgroup_fullname ~ "_My_Host_Group_*"
+----
+Replace _My_Host_Group_ with the name of the top-level host group.
```
Comment (maximiliankolb) [inline guides/common/modules/proc_executing-a-remote-job.adoc:23]:
```suggestion

Replace _My_Host_Group_ with the name of the top-level host group.
```

**Review by maximiliankolb — APPROVED** [review]
One small suggestion; rest LGTM!

File: guides/common/modules/proc_executing-a-remote-job.adoc (hunk 3 above)
Comment (adamlazik1) [reply guides/common/modules/proc_executing-a-remote-job.adoc:24]: Applied.

**Review by maximiliankolb — APPROVED** [review]

**Comment by adamlazik1** [comment]
Merged and cherry-picked:
   2ebbc8c7f..d93259a9f  3.5 -> 3.5
   faa3626f4..8fd3a9985  3.6 -> 3.6
   c9c9f4ec4..7a3b03179  3.7 -> 3.7
   26d20d214..bd0b7a095  3.8 -> 3.8

**Pull request merged**